website/src/css/font.scss

```diff
@@ -0,0 +1,109 @@
+// @font-face {
```

Comment:
**request**: Pleaes remove not needed code.

Reply:
👌